ideas/README.md

```diff
@@ -0,0 +1,141 @@
+# Ideas
+
+## Intro
+
+Welcome to the Google Summer of Code ideas repository!
+Here you can propose new ideas to participate to this fantastic event 
+and contribute to the nix/nixOS/nixpkgs community ecosystem!
+
+Below you will find all instructions to propose your ideas, so don't be shy
+to contribute!
+
+Summerily yours,
+the repo maintainers.
+
+## Submission fields explanation
+
+### ID
+The ID is composed by the current year and an incremental 4 digit integer
+assigned by one of the repo maintainers and will also be the directory name
+that will contain all the artifacts for that idea.
+
+### Title
+The title should be clear, concise and inspirative.
+
+It should also be prefixed by a categorey chosen among:
+    * nix
+    * nixpkgs
+    * nixos
+Depending on which area of the ecosystem more closely fits your idea.
+
+### Author
+Just the author's github handle.
+
+### Proposed Menthor
+If you're in contact with a member of the nix community who's interested in
+helping you as your direct menthor, you can mention the github handle here so
+that the repo maintainers wil be added to the conversation in the PR.
+
+### Effort
+This is the "size" of the effort level of the idea. It should be one of:
+    * SMALL
+    * MEDIUM
+    * LARGE
+
+### Difficulty
+This is how "hard" it is at a technical level to implement the idea. It can be:
+    * EASY
+    * MEDIUM
+    * HARD
```

Comment:
```suggestion
### Difficulty
This is how "hard" it is at a technical level to implement the idea. It can be:
 
- EASY
- MEDIUM
- HARD
```
same here with sticking to markdown conventions